automation_testing/tests/dataview/save_as.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { test, expect } from '@playwright/test';
+import { dataview } from '../../utils/data/dataview_data';
+import { DataviewPage } from '../../pages/DataViewPage';
+import { SaveAsComponent } from '../../pages/SaveAsComponent';
+
+test.describe('Data View', () => {
+
+    let dataviewPage: DataviewPage;
+    let saveAs: SaveAsComponent;
+    test.beforeEach(async ({ page }) => {
+        dataviewPage = new DataviewPage(page);
+        saveAs = dataviewPage.saveAsComponent;
+
+        await dataviewPage.visit();
+    });
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9498610)
```suggestion
	});
```